packaging/windows/WIXInstallers/KeybaseBundle/Bundle.wxs

```diff
@@ -48,11 +50,6 @@
                  Win64="yes"
                  />
 
-    <bal:Condition
-      Message="Please uninstall any previous versions of the Dokan driver package before continuing.">
-      <![CDATA[NOT InnoUninstallString]]>
-    </bal:Condition>
-
```

Comment:
also cleanup. InnoUninstallString occurred nowhere else; its job is done a different way now.